src/SFML/Graphics/RenderTarget.cpp

```diff
@@ -53,6 +53,9 @@ namespace
             case sf::BlendMode::DstAlpha:         return GL_DST_ALPHA;
             case sf::BlendMode::OneMinusDstAlpha: return GL_ONE_MINUS_DST_ALPHA;
         }
+        assert(false);
+        sf::err() << "Invalid value for sf::BlendMode::Factor! Fallback to GL_ZERO." << std::endl;
```

Comment:
Susnux is right, because this function is in the anonymous namespace only, so it's not nested in the `sf` namespace.

About the position of the `assert()`: Guess this is open to debate, since the failing assert should direct your right to the error message anyway. Actually having the error message as well might give someone the impression that the assert happened after that error (but yeah, both ways have their pros and cons).